banzai_floyds/frames.py

```diff
@@ -88,17 +94,68 @@ def get_output_data_products(self, runtime_context):
         else:
             return super().get_output_data_products(runtime_context)
 
+    def save_processing_metadata(self, context):
+        super().save_processing_metadata(context)
+        if 'REDUCER' not in self.meta:
+            self.meta['REDUCER'] = 'BANZAI'
+
     @property
     def profile(self):
         return self['PROFILE'].data
 
     @profile.setter
     def profile(self, value):
```

Comment:
I have changed how we load previous profile fits. Most of this logic existed already, but was spread across multiple files. I centralized it here.